python/gsw/main.py

```diff
@@ -0,0 +1,370 @@
+from flask import Flask, request, jsonify
+from flasgger import Swagger
+from flasgger.utils import swag_from
+from elasticsearch import Elasticsearch
+from datetime import datetime
+import threading
+import json
+import imaplib
+import base64
+import os
+import email
+import time
+import logging
+
+app = Flask(__name__)
+app.config["SWAGGER"]={"title": "PAN Ground Software", "uiversion": 2}
```

Comment:
Can you move these two lines down closer to the `if __name__ == "__main__"` line? Preferably, it should all be within the if-block. Right now you're alternating between defining classes/methods and instantiating them; it's easier to read if we define all the stuff first, then instantiate it.

Reply:
I can move it down to just before I declare the endpoints and the SwaggerUI API, but not after, otherwise I'll get an error.

Reply:
That makes sense.